spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -79,6 +82,8 @@ public GcpPubSubAutoConfiguration(GcpPubSubProperties gcpPubSubProperties,
 		this.finalCredentialsProvider = gcpPubSubProperties.getCredentials().hasKey()
 				? new DefaultCredentialsProvider(gcpPubSubProperties)
 				: credentialsProvider;
+		this.trustedPackages = gcpPubSubProperties.getTrustedPackages()
```

Comment:
Inline this in the `pubSubTemplate()` method.

Reply:
Will do. Somehow I didn't see we already have a `this.gcpPubSubProperties` haha